### PR TITLE
Update content

### PR DESCRIPTION
**Pull request by karlchillmaid**
After discussing with Ale, we’re going to update the Thanks page content so that it matches the commitment we make on the Support page.

### CODE REVIEW

File: app/templates/views/support/thanks.html [hunk 1]
```diff
@@ -22,14 +22,14 @@
           {% if out_of_hours %}
             We’ll reply within one working day.
           {% else %}
-            We’ll read your message in the next 30 minutes and reply within one
+            We’ll aim to read your message in the next 30 minutes and reply within one
```
Comment (karlchillmaid) [inline app/templates/views/support/thanks.html:25]:
```suggestion
            We’ll aim to read your message in the next 30 minutes and we’ll reply within one
```

File: app/templates/views/support/thanks.html [hunk 2]
```diff
@@ -22,14 +22,14 @@
...
-            We’ll read your message in the next 30 minutes and reply within one
+            We’ll aim to read your message in the next 30 minutes and we’ll reply within one
             working day.
           {% endif %}
         {% else %}
           {% if out_of_hours %}
             We’ll read your message when we’re back in the office.
           {% else %}
-            We’ll read your message in the next 30 minutes.
+            We’ll aim to read your message in the next 30 minutes.
```
Comment (karlchillmaid) [inline app/templates/views/support/thanks.html:32]: Can the reviewer please double check the scenario in which users see this content?

Comment (benthorner) [reply app/templates/views/support/thanks.html:32]: This is for [anonymous tickets](https://github.com/alphagov/notifications-admin/pull/3991/files#diff-8580ead6559100907653c52ac77a0304e95c02e412667f87700964617c6754c7R658). Do you want to proceed with the change?

Comment (karlchillmaid) [reply app/templates/views/support/thanks.html:32]: If it’s anonymous tickets in office hours, then yes please.

**Review by benthorner — APPROVED** [review]

**Pull request merged**